bench/workgen/runner/prepare_stress.py

```diff
@@ -0,0 +1,206 @@
+
+# Insert operations with snapshot isolation level and prepare_timestamp.
+ops = Operation(Operation.OP_INSERT, tables[0])
+ops = op_multi_table(ops, tables, False)
+ops = op_log_like(ops, log_table, 0)
+write_txn = txn(ops, 'isolation=snapshot')
+# use_prepare_timestamp - Commit the transaction with prepare, commit and durable timestamp.
+write_txn.transaction.use_prepare_timestamp = True
+thread1 = Thread(write_txn)
+# Thread.options.session_config - Session configuration.
+thread1.options.session_config="isolation=snapshot"
+# These operations include log_like operations, which will increase the number
+# of insert/update operations by a factor of 2.0. This may cause the
+# actual operations performed to be above the throttle.
+
+# Insert operations with snapshot isolation level and sets commit timestamp.
+ops = Operation(Operation.OP_UPDATE, tables[0])
+ops = op_multi_table(ops, tables, False)
+ops = op_log_like(ops, log_table, 0)
+update_txn = txn(ops, 'isolation=snapshot')
+# use_commit_timestamp - Commit the transaction with commit_timestamp.
+update_txn.transaction.use_commit_timestamp = True
+thread2 = Thread(update_txn)
+# Thread.options.session_config - Session configuration.
+thread2.options.session_config="isolation=snapshot"
+# These operations include log_like operations, which will increase the number
+# of insert/update operations by a factor of 2.0. This may cause the
+# actual operations performed to be above the throttle.
+thread2.options.throttle=500
+thread2.options.throttle_burst=1.0
+
+# Long running transactions. There is a 0.1 second sleep after a series of search and update
+# operations. The sleep op is repeated 10000 times and this will make these transcations to at
+# least run for ~17 minutes.
+search_op = Operation(Operation.OP_SEARCH, tables[0], Key(Key.KEYGEN_PARETO, 0, ParetoOptions(1)))
+update_op = Operation(Operation.OP_UPDATE, tables[0], Key(Key.KEYGEN_PARETO, 0, ParetoOptions(1)))
+ops = txn(((search_op + update_op) * 1000 + sleep(0.1)) * 10000)
+thread3 = Thread(ops)
+
+ops = Operation(Operation.OP_SLEEP, "0.1") + \
+      Operation(Operation.OP_LOG_FLUSH, "")
+logging_thread = Thread(ops)
+
+workload = Workload(context, 50 * thread0 + 50 * thread1 +\
+                    10 * thread2 + 100 * thread3 + logging_thread)
+workload.options.report_interval=5
+workload.options.run_time=500
+workload.options.max_latency=50000
+# oldest_timestamp_lag - Number of seconds lag to the oldest_timestamp from current time.
+workload.options.oldest_timestamp_lag=30
+# stable_timestamp_lag - Number of seconds lag to the stable_timestamp from current time.
+workload.options.stable_timestamp_lag=10
+# timestamp_advance is the number of seconds to wait before moving oldest and stable timestamp.
+workload.options.timestamp_advance=1
+workload.run(conn)
+
+end_time = time.time()
+run_time = end_time - start_time
+
+print('Workload took %d minutes' %(run_time//60))
+
+latency_filename = "WT_TEST" + "/latency.out"
```

Comment:
Instead of this, use
```
latency_filename = os.path.join(context.args.home, "latency.out")
```
`context.args.home` is preferred because with recent workgen changes, we can change the home directory using the `--home` argument when running the python script.  But if we hardcode the home directory here, that wouldn't work.  Also in Python, it's best to use `os.path.join` to concatenate path names, it is portable to other systems that use different path name conventions.